android/build.gradle

```diff
@@ -38,6 +38,6 @@ android {
 }
 
 dependencies {
-  implementation 'com.rudderstack.android.sdk:core:[1.0.22,)'
+  implementation 'com.rudderstack.android.sdk:core:1.2.1+'
```

Comment:
Hey @savy-91 we need to change here as well.

Reply:
I will change it but honestly thing that these changes are not related to my PR and you should apply them to a branch of yours and then ask me to rebase on it.